src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java

```diff
@@ -376,25 +376,25 @@ public String toString() {
      * List of keys that are deprecated
      */
     public static final List<String> DEPRECATED_KEYS = ImmutableList.of(
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_REST,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_REST_CATEGORIES,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_ENABLE_TRANSPORT,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_DISABLED_TRANSPORT_CATEGORIES,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_LOG_REQUEST_BODY,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_INDICES,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_EXCLUDE_SENSITIVE_HEADERS,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_BULK_REQUESTS,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_USERS,
-            ConfigConstants.OPENDISTRO_SECURITY_AUDIT_IGNORE_REQUESTS,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_INTERNAL_CONFIG_ENABLED,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_EXTERNAL_CONFIG_ENABLED,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_METADATA_ONLY,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_IGNORE_USERS,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_WATCHED_FIELDS,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_WRITE_METADATA_ONLY,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_WRITE_LOG_DIFFS,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_WRITE_IGNORE_USERS,
-            ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_WRITE_WATCHED_INDICES
+            ConfigConstants.SECURITY_AUDIT_ENABLE_REST,
```

Comment:
This introduces new key and immediately deprecates it. All those keys are already deprecated and should not be migrated to new keys.